cmd/guaccollect/cmd/files.go

```diff
@@ -44,6 +45,8 @@ type filesOptions struct {
 	blobAddr string
 	// poll location
 	poll bool
+	// use blob URL for origin instead of source URL
+	useBlobURL bool
```

Comment:
we should add to the comment here saying that if the blob store is persistent and we want to store the blob store source location.